ci/unit.yml

```
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.15.6
+    tag: latest
```

Please fix the dependencies using the latest (non-`x.y.0` docker image) as per https://github.com/ONSdigital/dp/blob/main/standards/DEPENDENCY_UPGRADING.md#build-and-test-version-docker-image
```suggestion
    tag: 1.16.7
```